bats/tests/preferences/surface-invalid-args.bats

```diff
@@ -0,0 +1,61 @@
+load '../helpers/load'
+
+local_setup() {
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+can_set_vm_to_vz() {
+    if ! is_macos; then
+        false
+    else
```

Comment:
I would structure the function something like this:

```shell
supports_vz_emulation() {
  if is_macos; then
    ...
    if ...; then
      return 0
    fi
  fi
  return 1
}
```

Having these `true` and `false` statements inside a big block, and relying on them being the last statement executed in the functions makes me nervous...